int8 init_joint1
int8 init_joint2
int8 init_joint3
int8 init_joint4
int8 init_joint5
int8 init_joint6
int8 save_init_pose
int8 arm_look
int8 arm_home


